net/dns/wsl_windows.go

```diff
@@ -228,8 +228,8 @@ func wslRun(cmd *exec.Cmd) (err error) {
 	}
 
 	cmd.SysProcAttr = &syscall.SysProcAttr{
-		Token:      syscall.Token(token),
-		HideWindow: true,
+		CreationFlags: windows.DETACHED_PROCESS,
```

Comment:
More specifically, given the WSL2 process hierarchy, I would not expect `stdout` redirection (and hence functions like `wslCombinedOutput`) to work.

Reply:
That's fair. I'll go with `CREATE_NO_WINDOW` for now, since our WSL stuff is pretty much disabled atm anyway. I like the idea of going with a pseudoconsole in the future.